### ISSUE TITLE
AM2 Crash when casting spell or going near Celestial Prism

### ISSUE THREAD

**Issue by InsaneChemist** [DrasticDemise/Hubris#23]
Recently, I have been unable to cast any AM2 spell that gives a buff or go near any of the etherium generating structures. It causes a crash to desktop that can only be fixed by deleting the potion effects manually.  This also includes boss debuffs such as watery grave. This is a new change, as it had no problem for most of the game, the only change I am aware of is that I attained AM2 level 20. I can attach the crash log if I need to, but I am not sure how useful it will be. I am playing on a LAN server, and the other person is not crashing, but he is AM2 level 6. The FAQ mentions a ticking entity crash, but this appears to be a different issue.

**Comment by InsaneChemist** [comment]
http://pastebin.com/Yc0FD5tj

Crashlog.

**Comment by DrasticDemise** [comment]
This log mentions adventure backpack and GUI's the most, are you wearing a backpack at the time?

**Comment by InsaneChemist** [comment]
Nope. The console says something other than he crash log, which is interesting
I was using dragon backpack much earlier in the mod, but I took it off well before this issue started. Also, just in case that was the problem, I just put the dragon backpack back on, and it still crashed when going near a celestial prism.

**Comment by InsaneChemist** [comment]
I tested removing the Adventure backpacks mod, there were no issues initially. When going near a celestial prism, I crashed with a different crash log.

http://pastebin.com/AfHqUds7

After some testing, it appears that an AM2 spell effect causes a crash when the icon related to it is displayed. First, AdventureBackpacks crashed because of the spell icon display at the top left (by default) and when that was removed, another mod caused a crash when the icon was displayed in the inventory screen.
